src/Faker/Provider/ps_AF/Text.php

```diff
@@ -0,0 +1,251 @@
+<?php
+
+namespace Faker\Provider\ps_AF;
+
+class Text extends \Faker\Provider\Text
+{
+    /**
+     * generates text string in Pashto
+     *
+     * @example 'په جمله کې ټاکندویه عناصر کېدای شي چې صفت وي، مضاف وي یا شمېر نوم'
+     * @param  integer                   $maxNbChars
+     * @param  integer                   $indexSize
+     * @return string
+     * @throws \InvalidArgumentException
+     */
+    public function realText($maxNbChars = 200, $indexSize = 2)
+    {
+        if ($maxNbChars < 10) {
+            throw new \InvalidArgumentException('maxNbChars must be at least 10');
+        }
+
+        if ($indexSize < 1) {
+            throw new \InvalidArgumentException('indexSize must be at least 1');
+        }
+
+        if ($indexSize > 5) {
+            throw new \InvalidArgumentException('indexSize must be at most 5');
+        }
+
+        $words = $this->getConsecutiveWords($indexSize);
+        $result = array();
+        $resultLength = 0;
+        // take a random starting point
+        $next = static::randomKey($words);
+        while ($resultLength < $maxNbChars && isset($words[$next])) {
+            // fetch a random word to append
+            $word = static::randomElement($words[$next]);
+
+            // calculate next index
+            $currentWords = explode(' ', $next);
+
+            $currentWords[] = $word;
+            array_shift($currentWords);
+            $next = implode(' ', $currentWords);
+
+            if ($resultLength == 0 && !preg_match('/^[\x{0600}-\x{06FF}]/u', $word)) {
+                continue;
+            }
+            // append the element
+            $result[] = $word;
+            $resultLength += strlen($word) + 1;
+        }
+
+        // remove the element that caused the text to overflow
+        array_pop($result);
+
+        // build result
+        $result = implode(' ', $result);
+
+        return $result.'.';
+    }
+
+    /**
+     * License: Creative Commons Attribution-ShareAlike License
+     *
+     * Language: Pashto
+     *
+     * @see https://www.bbc.co.uk/academy/ps/articles/art20131031113816478
+     * @var string
+     */
+    protected static $baseText = <<<EOD
```

Comment:
Can we get rid of all the new lines? Also the data is not open

Reply:
@pimjansen Can you please check now?